packages/wallets/src/components/Base/WalletTextField/WalletTextField.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { ChangeEvent, ComponentProps, CSSProperties, FC, useState } from 'react';
+import React, { ChangeEvent, ComponentProps, FC, useState } from 'react';
 import HelperMessage, { HelperMessageProps } from './HelperMessage';
 import './WalletTextField.scss';
 
 export interface WalletTextFieldProps extends ComponentProps<'input'>, HelperMessageProps {
     defaultValue?: string;
     label?: string;
-    maxWidth?: CSSProperties['maxWidth'];
```

Comment:
Might need to refactor `ResubmitPOA` also

Reply:
Updated